INSTALL.md

```diff
@@ -56,3 +56,44 @@ dokku$ sudo mkdir -p /var/log/journal
 dokku$ dokku enter job-server
 container$ python manage.py sendtestemail me@myemail.org
 ```
+
+## Rotating the Django secret key
+
+If this is a *routine rotation* of the secret key
+— and not a compromised secret key —
+you can temporarily leave the secret key to be rotated alongside the new secret key.
+
+This allows for changing the secret key without logging out all users at once.
+
+There are three steps to this process.
+
+### 1. Duplicate the existing secret key
+
+```bash
+dokku config:get job-server SECRET_KEY
+dokku config:set job-server OLD_SECRET_KEY="$(dokku config:get job-server SECRET_KEY)"
```

Comment:
I may have misunderstood, but doesn't the get in the second line make the get in the first line unnecessary?

Reply:
Good spot; that's correct and I'll remove the first of the two lines.